packages/plugins/Toast/src/store/index.ts

```diff
@@ -48,9 +48,15 @@ const storeModule: PolarModule<ToastState, ToastState> = {
       }
       commit('addToast', smearedToast)
       if (toast.timeout) {
-        setTimeout(() => {
-          commit('removeToast', smearedToast)
-        }, toast.timeout)
+        if (toast.type !== 'error') {
+          setTimeout(() => {
+            commit('removeToast', smearedToast)
+          }, toast.timeout)
+        } else {
+          console.warn(
+            'Timeouts for error messages are disabled, they can only be dismissed manually.'
```

Comment:
```suggestion
            '@polar/plugin.toast: Timeouts for error messages are disabled, they can only be dismissed manually.'
```
I'd also be open for just `Toast`, but this is probably a clearer indication